### PR TITLE
FFM-7293, FFM-7296: relnotes Node 1.2.13/4

### PR DESCRIPTION
**Pull request by StrawberryPad**
For 2 same-day releases of Node.js SDK

### CODE REVIEW

File: release-notes/feature-flags.md [hunk 1]
```diff
@@ -7,12 +7,53 @@ sidebar_position: 6
...
+
+This release does not include early access features.
+
+### Fixed issues
+
+#### Feature Flags SDKs
+
+The **Node.js server SDK** has been updated to version **1.2.14** with the following updates:
+
+* The event source was updated to version 2.1.3. (FFM-7198)
```
Comment (knagurski) [inline release-notes/feature-flags.md:32]:
```suggestion
* The EventSource library was updated to version 2.1.3. (FFM-7198)
```

**Review by erdirowlands — APPROVED** [review]
Looks good!

**Review by erdirowlands — DISMISSED** [review]

File: release-notes/feature-flags.md [hunk 2]
```diff
@@ -7,12 +7,53 @@ sidebar_position: 6
...
+
+### Fixed issues
+
+#### Feature Flags SDKs
+
+The **Node.js server SDK** has been updated to version **1.2.14** with the following updates:
+
+* The EventSource library was updated to version 2.1.3. (FFM-7198)
+* On streaming errors, the error was not included in the `retrying` event payload. This fix adds the error to the payload. (FFM-7198)
+* Checks were added to see if errors are retryable, and if not, the SDK stops retrying. (FFM-7198)
```
Comment (aimurphy) [inline release-notes/feature-flags.md:34]:
```suggestion
* Added checks to see if errors can be retried, and if they can't, then the SDK stops retrying. (FFM-7198)
```

I know devops involves these type of terms, like retryable, but I am not sure how well it translates for ESL readers.

**Review by knagurski — APPROVED** [review]
🔥

**Review by aimurphy — APPROVED** [review]
Optional suggestions, otherwise LGTM

**Pull request merged**